### PR TITLE
Create volume-of-cylinder.jl

### LINKED ISSUE #1035: Write a Julia programme to find volume of cylinder
### Description

Write a programme to find volume of cylinder

Volume of cylinder = π * (radius)^2 * height

```
radius : 2
height : 3

volume : 37.69911
```

### PR DESCRIPTION
**Pull request by SpirosArk**
### Why:

Closes #1035 

### Check off the following:

- [x ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x ] For content changes, I have completed the [self-review checklist](https://github.com/codinasion/codinasion/blob/master/contributing/self-review.md).

### CODE REVIEW

File: programme/volume-of-cylinder/volume-of-cylinder.jl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Program to calculate volumeOfCylinder
```
Comment (KikiManjaro) [inline programme/volume-of-cylinder/volume-of-cylinder.jl:1]:
```suggestion
#Programme to calculate volumeOfCylinder
```

**Review by KikiManjaro — CHANGES_REQUESTED** [review]
Minor fixes related to #1295

**Comment by SpirosArk** [comment]
I committed changes please re review

**Review by KikiManjaro — APPROVED** [review]

**Review by harshraj8843 — APPROVED** [review]
👍🏻

**Pull request merged**